src/antsibull_changelog/plugins.py

```diff
@@ -433,8 +433,7 @@ def _get_ansible_core_version(paths: PathsConfig) -> packaging.version.Version:
         match = re.match(regex, output)
         if match:
             return packaging.version.Version(match.group(1))
-    # pylint:disable-next=broad-exception-raised
-    raise Exception(
+    raise ValueError(
```

Comment:
This feels more like a RuntimeError to me. [ValueError](https://docs.python.org/3/library/exceptions.html#ValueError) is used for invalid input provided by a {C,A}PI user.

Reply:
Good idea! Adjusted in 74727958c25b79159f6c404e442a345d492e9f2e.